tools.rst

```diff
@@ -19,15 +19,19 @@ We ask everyone to assess themselves with regards to using the tools - Do you ha
 
 Inductions are required on equipment that is potentially very dangerous or that is expensive. At present an induction is required before members can use the following equipment:
 
-* 3D printers
+
 * Laser cutter
-* Metal lathe
-* Metal milling machine
-* Tig welding equipment
```

Comment:
why was tig removed? its still needed

Reply:
oops - missed that, added it back.  thanks for spotting